src/backproject.jl

```diff
@@ -3,153 +3,117 @@
 export backproject, backproject!
 
 """
-    backproject!(image, view, plan, workarray, viewidx)
+    backproject!(image, view, plan, viewidx)
 Backproject a single view.
 """
 function backproject!(
     image::AbstractArray{<:RealU, 3},
     view::AbstractMatrix{<:RealU},
     plan::SPECTplan,
-    workarray::Vector{Workarray},
-    viewidx::Int
+    viewidx::Int,
 )
 
     Threads.@threads for z = 1:plan.imgsize[3] # 1:nz
```

Comment:
i would prefer to have this loop inside the imrotate code eventually.
But OK for now.